### ISSUE TITLE
Next button on ToolsAndEulaPage gives persistent spurious error

### ISSUE THREAD

**Issue by ranasaria** [microsoft/azuredatastudio#13719]
**Steps to Reproduce:**

1. Start with a state where az.cmd is not installed
2. Kickoff BDC deployment wizard
3. The tools and eula page warns that az.cmd is not installed.
4. Click 'Install Tools' button to install az.cmd. Click 'Next' once the installation is done.

**Actual:** An unexpected error 'Some tools are still not installed' shows up. This error persists if the user dismisses the error and clicks 'Next'. The wizard is rendered to be unusable.
![image](https://user-images.githubusercontent.com/41588310/101426591-228e0900-38b2-11eb-8875-bc7bec8969dc.png)

**Expected:** The wizard proceeds to the next page.

**Workaround:** Cancel out of the wizard and launch it again. All tools are discovered and the wizard proceeds to 2nd page.

**Azure Data Studio Version**:
```
Version: 1.25.0-rc1
Commit: 2d50d2c5d1ed5f426c46e06cee436dce8debc6c9
Date: 2020-12-03T02:46:44.284Z
VS Code: 1.48.0
Electron: 9.3.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.19042
```

**Comment by Charles-Gagnon** [comment]
Duplicate of https://github.com/microsoft/azuredatastudio/issues/13679
